src/generated/resources/data/forge/tags/blocks/ores/singular.json

```diff
@@ -0,0 +1,12 @@
+{
+  "replace": false,
+  "values": [
+    "minecraft:ancient_debris",
+    "minecraft:coal_ore",
+    "minecraft:diamond_ore",
+    "minecraft:emerald_ore",
+    "minecraft:gold_ore",
+    "minecraft:iron_ore",
```

Comment:
Likewise with dense ores, should deepslate variants be included? Or does it not matter since there's going to be a `ores/in_` tag anyway that would separate this?